certdb/db_config.go

```diff
@@ -0,0 +1,59 @@
+package certdb
+
+import (
+	"database/sql"
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+
+	cferr "github.com/cloudflare/cfssl/errors"
+	"github.com/cloudflare/cfssl/log"
+	_ "github.com/lib/pq"           // import just to initialize Postgres
+	_ "github.com/mattn/go-sqlite3" // import just to initialize SQLite
+)
+
+// DBConfig contains the database driver name and configuration to be passed to Open
+type DBConfig struct {
+	DriverName     string
```

Comment:
nit: add json annotation to those fields, and make an example json that can be converted to DBConfig via LoadFile(), well yes I am asking for a test file for db_config. :)